cmd/libs/go2idl/deepcopy-gen/generators/deepcopy.go

```diff
@@ -384,8 +385,12 @@ func (g *genDeepCopy) Init(c *generator.Context, w io.Writer) error {
 	sw.Do("if err := $.scheme|raw$.AddGeneratedDeepCopyFuncs(\n", map[string]interface{}{
 		"scheme": scheme,
 	})
+	reflect := c.Universe.Type(types.Name{Package: "reflect", Name: "TypeOf"})
+	runtime := c.Universe.Type(types.Name{Package: runtimePackagePath, Name: "GeneratedDeepCopyFunc"})
 	for _, t := range g.typesForInit {
-		sw.Do(fmt.Sprintf("%s,\n", g.funcNameTmpl(t)), argsFromType(t))
+		g.imports.AddType(reflect)
+		g.imports.AddType(runtime)
+		sw.Do(fmt.Sprintf("runtime.GeneratedDeepCopyFunc{Fn: %s, InType: reflect.TypeOf(func() *$.type|raw$ {var x *$.type|raw$; return x}())},\n", g.funcNameTmpl(t)), argsFromType(t))
```

Comment:
I mis-parsed this.  Looking furtther, I realize my concern about registration does not apply here because this IS registration :)